libraries/src/main/java/com/baeldung/circularfifoqueue/example/CircularFifoQueueExample.java

```diff
@@ -0,0 +1,101 @@
+package com.baeldung.circularfifoqueue.example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.collections4.queue.CircularFifoQueue;
+
+public class CircularFifoQueueExample {
```

Comment:
Why do we have this helper class? The test cases would be much clearer to a new reader if we simply did all of the work in the test class.